build.sh

```diff
@@ -16,5 +27,5 @@ mkdir -p $P/scratch;
 cd $P/scratch;
 rm -rf $P/install;
 $SRC/configure --prefix=$P/install --with-mountutildir=$P/install/sbin --with-initdir=$P/install/etc --localstatedir=/var --enable-bd-xlator=yes --enable-debug --silent
-make install CFLAGS="-g -O0 -Wall -Werror" -j 4 >/dev/null
+make install CFLAGS="-g -O0 -Wall -Werror" -j ${nproc} >/dev/null
```

Comment:
According to the man page, -j without an option will not constrain the number of jobs, which I think means it will just spawn as many as it can based on dependencies.  Not sure if that's a "suitable" number for our purposes or not.